org.eclipse.lsp4e/plugin.xml

```diff
@@ -350,6 +376,42 @@
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
             sequence="Ctrl+Alt+H">
       </key>
+      
+      <!-- edit -->
+      <key
+            sequence="M2+M3+ARROW_UP"
+            contextId="org.eclipse.ui.genericeditor.genericEditorContext"
+            commandId="org.eclipse.lsp4e.selectionRange.up"
+            schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"/>
+      <key
+            platform="carbon"
+            sequence="M2+M3+ARROW_UP"
+            contextId="org.eclipse.ui.genericeditor.genericEditorContext"
+            commandId=""
+            schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"/>
+      <key
+            platform="carbon"
+            sequence="CTRL+SHIFT+PAGE_UP"
+            contextId="org.eclipse.ui.genericeditor.genericEditorContext"
+            commandId="org.eclipse.lsp4e.selectionRange.up"
+            schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"/>
```

Comment:
carbon isn't supported any more, let's remove.

Reply:
removed (and for rename too)